akka-stream/src/main/scala/akka/stream/SslTlsOptions.scala

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (C) 2015-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream
```

Comment:
Yes, and a `akka.stream` package for the shared things—that is a difference to how HTTP works. We don’t have separate Materializer types for Java and Scala, for example. Just like we don’t see any need to use different TLSClientAuth types; what would be the benefit?